server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -571,18 +572,25 @@ export class AssetService {
   }
 
   private getThumbnailPath(asset: AssetEntity, format: GetAssetThumbnailFormatEnum) {
+    if (!Object.values(GetAssetThumbnailFormatEnum).includes(format)) {
+      throw new UnsupportedMediaTypeException(`Unsupported thumbnail format requested: ${format}`);
+    }
+
```

Comment:
We should be able to replace this with class validation in the dto directly btw. Maybe that is for a separate PR though.

Reply:
Good idea! Done